stable_horde.py

```diff
@@ -376,7 +384,7 @@ async def submit(self, id: str, seed: str, generation: str):
         if (r.status == 200 and res.get("reward") is not None):
             self.state.status = f"Submission accepted, reward {res['reward']} received."
         elif (r.status == 400):
-            self.state.status = "ERROR: Generation Already Submitted"
+            self.state.status = "ERROR Submitting: {res}"
```

Comment:
I think it would be better to "extract" the message from the backend instead of the whole response json object.

Reply:
Agreed, I just didn't know the format so something was better then nothing to start with.